### PR TITLE
Multiple images

### PR DESCRIPTION
**Pull request by SrinandhiniM**
Added a column for adding multiple images in the post and displaying the post in carousel View.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -106,14 +109,15 @@ def auth_subscriber
...
   def post_values
-    params.require(:post).permit(:title, :body, :saved, :is_drafted, :closed, :community_id)
+    params.require(:post).permit(:title, :body, :saved, :is_drafted, :closed, :community_id, images: [])
   end
 
   def find_my_communities
     @subscriptions = Subscription.where(account_id: current_account.id)
     @my_communities = []
     @subscriptions.each do |subscription|
       @my_communities << Community.find(subscription.community_id)
```
Comment (Venirails) [inline app/controllers/posts_controller.rb:119]: please align this properly

File: app/views/posts/_form.html.erb [hunk 2]
```diff
@@ -14,6 +14,7 @@
       <div class="form-group">
         <%= label_tag :body %>
         <%= form.rich_text_area :body, class: "textarea" %>
+        <%= form.file_field :images, multiple: true, :class=>"form-control", :placeholder=>"" %>
```
Comment (Venirails) [inline app/views/posts/_form.html.erb:17]: change this syntax like
 class: "form-control"

 I mentioned in slack

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -6,11 +6,39 @@
       <div class="fa fa-arrow-down downvote <%= is_downvoted @post %>"></div>
     </div>
   </div>
-  <div class="col-11">
+  
+  <div class="col-9" >
+  <div class="card" class="card-textarea">
+   <div class="card-body" class="card-textarea">
     <h2><%= @post.title %></h2>
```
Comment (Venirails) [inline app/views/posts/show.html.erb:14]: remove unwanted spaces

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (SrinandhiniM) [reply app/controllers/posts_controller.rb:119]: Aligned Now

**Comment by iMagesh** [comment]
@SrinandhiniM you need to resolve the conflicts now

**Comment by SrinandhiniM** [comment]
All conflicts resolved now

**Comment by iMagesh** [comment]
@SrinandhiniM no it still shows conflicts. can you see it below 👇
In this PR

**Pull request merged**